<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>930</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>930</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CalendarV1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCalendarWidget" name="CalendarWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>521</width>
      <height>401</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>170</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>AddActivity</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>170</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>RemoveActivity</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>170</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit Activity</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>30</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Nome </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>70</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Descrizione</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>100</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Ora inizio</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>130</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Ora fine</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NameLine">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>40</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="DescName">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>70</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QTimeEdit" name="StartTimeEdit">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>100</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QTimeEdit" name="EndTimeEdit">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>130</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>545</x>
      <y>210</y>
      <width>361</width>
      <height>221</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>634</x>
      <y>10</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Aggiungi una nuova attivit√†</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>930</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuImpostazioni">
    <property name="title">
     <string>Impostazioni</string>
    </property>
    <widget class="QMenu" name="menuImpostazioni_Date_e_Ora">
     <property name="title">
      <string>Impostazioni Date e Ora</string>
     </property>
     <addaction name="separator"/>
     <addaction name="actionCambia_fuso_orario"/>
     <addaction name="actionImposta_data"/>
    </widget>
    <addaction name="menuImpostazioni_Date_e_Ora"/>
   </widget>
   <addaction name="menuImpostazioni"/>
  </widget>
  <action name="actionCambia_fuso_orario">
   <property name="text">
    <string>Cambia fuso orario</string>
   </property>
  </action>
  <action name="actionImposta_data">
   <property name="text">
    <string>Imposta data </string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
